src/assets/stylesheets/avatar-editor.scss

```diff
@@ -107,7 +107,7 @@
       }
       a {
         margin-left: 10px;
-        color: $action-label-color;
```

Comment:
Why is this variable is resolved at build time instead of runtime? (Seems to be done this way for all usages of `action-label-color` so guessing its intentional.)

Reply:
the `$`'s are used because those macros expand into vars that have default values. so the alternative would have been to paste the default values at every site, which isn't DRY